.travis/yapf.sh

```diff
@@ -9,14 +9,33 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-yapf \
-    --style "$ROOT/.style.yapf" \
-    --in-place --recursive --parallel \
-    --exclude 'python/ray/cloudpickle' \
-    --exclude 'python/ray/dataframe' \
-    --exclude 'python/ray/rllib' \
-    -- \
-    'test' 'python'
+lint() {
+    yapf \
+        --style "$ROOT/.style.yapf" \
+        --in-place --recursive --parallel \
+        -- \
+        "$@"
+}
+
+lint_all() {
+
+    yapf \
+        --style "$ROOT/.style.yapf" \
+        --in-place --recursive --parallel \
+        --exclude 'python/ray/cloudpickle' \
+        --exclude 'python/ray/dataframe' \
+        --exclude 'python/ray/rllib' \
+        -- \
+        test python
+}
+
+# This flag lints individual files. --files *must* be the first command line
+# arg to use this option.
+if [[ "$1" == '--files' ]]; then
```

Comment:
Why this change? --files isn't used below.

Reply:
This is a poor man's argparse since I hate messing with bash's getopts. If `--files` is the first argument, then only the files passed are formatted.